test/general/test_implemented.py

```diff
@@ -52,3 +52,68 @@ def test_slot_data(self):
     def test_no_failed_world_loads(self):
         if failed_world_loads:
             self.fail(f"The following worlds failed to load: {failed_world_loads}")
+
+    def test_explicit_indirect_conditions_spheres(self):
+        """Tests that worlds using explicit indirect conditions produce identical spheres as when using implicit
+        indirect conditions"""
+        # Because the iteration order of blocked_connections in CollectionState.update_reachable_regions() is
+        # nondeterministic, this test may sometimes pass with the same seed even when there are missing indirect
+        # conditions.
+        for game_name, world_type in AutoWorldRegister.world_types.items():
+            multiworld = setup_solo_multiworld(world_type)
+            world = multiworld.get_game_worlds(game_name)[0]
+            if not world.explicit_indirect_conditions:
+                # The world does not use explicit indirect conditions, so it can be skipped.
+                continue
+            # The world may override explicit_indirect_conditions as a property that cannot be set, so try modifying it.
+            try:
+                world.explicit_indirect_conditions = False
+                world.explicit_indirect_conditions = True
+            except Exception:
+                # Could not modify the attribute, so skip this world.
+                with self.subTest(game=game_name, skipped="world.explicit_indirect_conditions could not be set"):
+                    continue
```

Comment:
Asked about this in DMs but I'm still a little concerned about this exception since a world could override it to be `True` just to pass this test when it _should_ be failing it.